common/common_functions.sh

```diff
@@ -223,6 +223,7 @@ multus_install(){
 multus_configuration() {
     status=0
     echo "Configure Multus"
+    get_arch
```

Comment:
can get_arch be modified to return the ARCH instead of setting an environment variable ?
how it works today is not ideal.

then you just do local assingment

```
local ARCH=`get_arch`
```

or similar (not a bash expert)